test/node-api/test_threadsafe_function/test_uncaught_exception.js

```diff
@@ -1,9 +1,8 @@
 'use strict';
-// Flags: --force-node-api-uncaught-exceptions-policy
 
 const common = require('../../common');
 const assert = require('assert');
-const binding = require(`./build/${common.buildType}/binding`);
+const binding = require(`./build/${common.buildType}/test_uncaught_exception`);
 
 const callbackCheck = common.mustCall((err) => {
```

Comment:
Would it be possible to put this code in a common module? That would make it clear that the same test is supposed to work with both versions, and only the preprocessor flags and the node launch flags differ. I think files not starting with `test_` are not executed when the test suite runs.